include/constants/battle_config.h

```diff
@@ -92,6 +92,11 @@
     #define GEN_8 5
 #endif
 
+
+// Mega Evolution settings
+#define B_MEGA_EVO_ALTER_TURN_ORDER GEN_7 // In gen6, mega evolving doesn't change the turn order when mega evolving occurs. This is fixed in gen 7
```

Comment:
```suggestion
#define B_MEGA_EVO_TURN_ORDER GEN_7 // In Gen7, a Pokémon's Speed after Mega Evolution is used to determine turn order, not its Speed before.
```

Reply:
There's also an empty space right afterward that should be removed for consistency's sake.